### PR TITLE
Feature/add nvmrc file

### PR DESCRIPTION
**Pull request by psweeting1**
**Pull Request Description**
This PR introduces updates to the project’s Node.js version management and improves the documentation for setting up the environment.

**Changes Made**

Added `.nvmrc` File:

Introduced the .nvmrc file to specify the Node.js version required for this project.
This enables NVM (Node Version Manager) to automatically set the correct version of Node.js when working on the project.

Updated README:

Added instructions on how to download and install NVM.
Provided detailed guidance on how to use the .nvmrc file to set the project’s Node.js version using NVM.

Added Node.js Version Information in README:

Updated the README to outline the required versions for both Node.js and other dependencies needed to run the project.
This helps new contributors set up the correct environment quickly.

Updated package-lock.json:

Aligned the package-lock.json file with the current project’s Node.js management version and installed dependencies.
This ensures that the correct versions are locked and ensures consistent installation across environments.

**Why These Changes Are Necessary**

Version Consistency: The .nvmrc file ensures everyone is using the correct Node.js version, eliminating potential conflicts and inconsistencies in the development environment.

Improved Onboarding: The updated README now includes all necessary information for setting up the Node.js environment, making it easier for new developers to get started.

Package Integrity: Updating the package-lock.json ensures the project’s dependencies are consistent and aligned with the required versions for the current Node.js setup.

**How to Use the `.nvmrc` File**

Install NVM by following the instructions in the README.
Once NVM is installed, navigate to the project directory and run:
bash
Copy code
```
nvm install
nvm use
```
This will automatically switch your Node.js version to the one specified in the `.nvmrc` file.

**Testing**
Verified that NVM properly switches Node.js versions based on the `.nvmrc` file.
Ensured that the project runs as expected with the updated Node.js version and dependencies.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,12 @@ This is a template [Flask](https://flask.palletsprojects.com) app using the [GOV
 
 The app is provided intentionally bare, with just the essential parts that all services need, such as error pages, accessibility statement, cookie banner, cookie page and privacy notice. It uses a number of other packages to provide the [features](#features) described below with sensible and best-practice defaults. Please read the [next steps](#next-steps) section for guidance on how to start building out your app on top of this template.
 
+# Versioning 
+- Python: 3.12
+- Node.js: 22.12.0
+- npm: 10.9.0
+- Pip: 24.3.1
```
Comment (BenMillar-MOJ) [inline README.md:20]: pip should either be in full lowercase or full upper case, just like npm.

**Review by BenMillar-MOJ — CHANGES_REQUESTED** [review]
Looks great, thank you very much for adding this!

**Review by TawneeOwl — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (psweeting1) [reply README.md]: pip should be lowercase

**Pull request merged**